python/lsst/ts/ofc/ofc.py

```diff
@@ -76,13 +75,13 @@ class OFC:
     PSSN: Normalized point source sensitivity.
     """
 
-    def __init__(self, ofc_data, log=None):
+    def __init__(self, ofc_data: OFCData, log: logging.Logger = None) -> None:
         if log is None:
             self.log = logging.getLogger(type(self).__name__)
         else:
             self.log = log.getChild(type(self).__name__)
 
-        self.pssn_data = dict(sensor_id=None, pssn=None)
+        self.fwhm_data = None
```

Comment:
Aren't you doing this in  #54 ? I think this topic is a bit contentious no?

Reply:
Removed these changes from this PR